### DISCUSSION TITLE
Does rook support directly upgrading from 1.1 to 1.5, 1.1 to 1.6 or others?

### DISCUSSION THREAD

**Discussion by olivejing** [rook/rook#9887] [General]
From rook documents, it supports:
[Upgrade 1.5 to 1.6](https://rook.io/docs/rook/v1.6/ceph-upgrade.html)
[Upgrade 1.4 to 1.5](https://rook.io/docs/rook/v1.5/ceph-upgrade.html)
[Upgrade 1.3 to 1.4](https://rook.io/docs/rook/v1.4/ceph-upgrade.html)
[Upgrade 1.2 to 1.3](https://rook.io/docs/rook/v1.3/ceph-upgrade.html)
[Upgrade 1.1 to 1.2](https://rook.io/docs/rook/v1.2/ceph-upgrade.html)
[Upgrade 1.0 to 1.1](https://rook.io/docs/rook/v1.1/ceph-upgrade.html)
[Upgrade 0.9 to 1.0](https://rook.io/docs/rook/v1.0/ceph-upgrade.html)
[Upgrade 0.8 to 0.9](https://rook.io/docs/rook/v0.9/ceph-upgrade.html)
[Upgrade 0.7 to 0.8](https://rook.io/docs/rook/v0.8/upgrade.html)
[Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
[Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)

I want to know whether it supports directly upgrading from 1.1 to 1.5 or 1.6. 
My current version is 1.1, from the documents, i need to upgrade it to 1.2, then upgrade to 1.3, ... then upgrade to 1.6, I need to upgrade 5 times, right?  I hope to directly upgrade it to 1.6.

**Comment by parth-gr** [comment]
Probably you can only do a sequential jump of 1.

**Comment by travisn** [comment]
Correct, only one minor version upgrade is supported at a time. You might get lucky jumping versions, but I would not recommend it and it is not supported.

**Comment by olivejing** [comment]
Ok, thanks for your quickly response and support.
